src/Features/Core/Portable/FindUsages/DefinitionItem.DocumentLocationDefinitionItem.cs

```diff
@@ -50,7 +50,7 @@ public override bool CanNavigateTo(Workspace workspace)
                 return SourceSpans[0].CanNavigateTo();
             }
 
-            public override bool TryNavigateTo(Workspace workspace, bool isPreview)
+            public override bool TryNavigateTo(Workspace workspace, bool isPreview, bool activatePreview = false)
```

Comment:
can you make this non-optiona?  it will allow us to review all the callers to mae sure the're doing the right thing.

Reply:
Sure. The only other place I know that would need this is probably the external items called earlier. That said, I'm not sure what "IsExternal" means in this case?